src/Generator.Bind/Writers/BindingWriter.cs

```diff
@@ -142,7 +142,7 @@ private Task WriteWrappersAsync()
             var extensionNames = _profile.GetExtensions();
 
             // Create the interop setup necessary
-            var createConstructorFileTask = CreateWrapperClassConstructorFileAsync(wrappersOutputDirectory);
+            var createConstructorFileTask = Task.Run(() => CreateWrapperClassConstructorFileAsync(wrappersOutputDirectory));
```

Comment:
Why is Task.Run used here?

Reply:
The method name should no longer be Async. Will fix.